pynest/nest/tests/test_sp/test_disconnect.py

```diff
@@ -75,26 +77,26 @@ def test_synapse_deletion_one_to_one_no_sp(self):
                 conns = nest.GetConnections(
                     [neurons[0]], [neurons[2]], syn_model)
                 if mpi_test:
-                    connstotal = None
-                    conns = self.comm.allgather(conns, connstotal)
-                    conns = filter(None, conns)
+                    conns = self.comm.allgather(conns)
+                   # conns = filter(None, conns)
```

Comment:
Please remove the commented out code. This also applies to the occurences below.